views/list_all_documents_in_collection.aql

```diff
@@ -0,0 +1,6 @@
+// Return *all* full documents in a collection
+// Args:
+//   collection - name of collection to count docs
+
+for v in @@collection
+  return v
```

Comment:
We might want to put a limit on this so someone does not tie up Arango by running this on ws_object_versions. We also need to give another think on access control.

Reply:
The API will limit things. Query results are limited to 100 and more result pages can be fetched through a cursor. RAM usage for a single query is also capped. I think this should be fine in prod actually.

Reply:
But regarding access control -- yes. We'll have to update this for that.

Reply:
so `Return *all* full documents in a collection` is a LIE! Good to know about the caps on queries though.

Reply:
Not a lie if you follow the magic cursor!
Say you had 200 docs in a collection:
- POST /query {name: list_all_documents_in_collection} -> 100 results, cursor 123
- GET /cursor {id: 123} -> 100 next results, no cursor

Reply:
I might change the above API to make things a little less confusing:

* GET /query_results {view: lsit_all_documetns_blah} -> cursor 123
* GET /query_resutls {cursor_id: 123}